sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/SonarScanner.java

```diff
@@ -44,10 +47,33 @@
   }
 
   @Override
+  public SonarScanner setEnvironmentVariable(String name, String value) {
+    return (SonarScanner) super.setEnvironmentVariable(name, value);
+  }
+
+  /**
+   * @deprecated since 3.10 use {@link #setScannerVersion(String)}
+   */
+  @Deprecated
+  @Override
   public SonarScanner setRunnerVersion(String s) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)